bftengine/include/bftengine/DbCheckpointMetadata.hpp

```diff
@@ -53,10 +53,21 @@ struct DbCheckpointMetadata : public concord::serialize::SerializableFactory<DbC
     }
   };
   std::map<CheckpointId, DbCheckPointDescriptor> dbCheckPoints_;
+  uint64_t lastCmdSeqNum_{0};
+  TimeDuration lastCmdTimestamp_{0};
   DbCheckpointMetadata() = default;
-  void serializeDataMembers(std::ostream& outStream) const override { serialize(outStream, dbCheckPoints_); }
-  void deserializeDataMembers(std::istream& inStream) override { deserialize(inStream, dbCheckPoints_); }
+  void serializeDataMembers(std::ostream& outStream) const override {
+    serialize(outStream, lastCmdSeqNum_);
```

Comment:
I have a query,
Should we keep the lastCmdSeqNum_ and lastCmdTimestamp for all checkpoint. If yes then we also have to change the DB_CHECKPOINT_METADATA_MAX_SIZE
to  MAX_ALLOWED_CHECKPOINTS * (sizeof(std::pair<CheckpointId, DbCheckpointMetadata::DbCheckPointDescriptor>) +
    sizeof(DbCheckpointMetadata::lastCmdSeqNum_) + sizeof(DbCheckpointMetadata::lastCmdTimestamp_))

Reply:
yes, size of lastCmdSeqNum and lastCmdTimeStamp is also included. Please check the last line of this file